functions/src/repository/response-git-webhook-repository.ts

```diff
@@ -47,7 +48,7 @@ export const onResponseGitWebhookRepository: HttpsFunction = https.onRequest((re
 
     const inputData: any = req.body;
     let result: Promise<any>;
-
+    
```

Comment:
Tab not needed here, can you check your settings because these should be stripped when saving the file to avoid these types of changes

Reply:
I have removed it